.github/workflows/sast.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
     if: (github.repository_owner == 'contentful') && (endsWith(github.actor, '[bot]') == false)
     steps:
       - name: Clone repo
-        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4
```

Comment:
@karankohli-cf  Any reason we pin to the SHA instead of the version? Did we want to exclude github/actions from autoupdating?